### DISCUSSION TITLE
evernote-backup init-db

### DISCUSSION THREAD

**Discussion by Flemms** [vzhd1701/evernote-backup#76] [Q&A]
Hello

I have installed "pipx" using "scoop":
scoop install pipx
pipx ensurepath

I have installed "evernote-backup"  like this "pipx install evernote-backup"

I have downloaded and unzipped "Download the latest release"

Using the command line I run "evernote-backup.exe init-db"

After entering username the password prompt takes no keystrokes.

What do I do wrong?

regards, Flemming

**Comment by git-lara** [comment]
Same problem here.

**Comment by kraftskunk** [comment]
Too good to be true. This tool spits out an error at login. Creating a bug report after this post.

**Comment by chrowe** [comment]
Have you tried using the `--oauth` option? Per https://github.com/vzhd1701/evernote-backup/pull/109 I think this is now required for all login options.
